barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.hpp

```diff
@@ -72,6 +110,13 @@ struct AcirFormat {
         quad_constraints;
     std::vector<BlockConstraint> block_constraints;
 
+    // Number of gates added to the circuit per original opcode.
+    // Has length equal to num_acir_opcodes.
+    std::vector<size_t> gates_per_opcode = {};
```

Comment:
`= {}` and `Type{}` is in general not necessary*. (the exception is `FF` afaik, which is an interesting design decision)

*for allocation/default construction. Sometimes it's useful to force zero-initialization.

Reply:
Seems like I need to do this because else I need to add it to every constructor expression for AcirFormat